.storybook-sdk/preview.tsx

```diff
@@ -27,6 +27,18 @@ export const globalTypes: GlobalTypes = {
       icon: "paintbrush",
       items: storybookThemeOptions,
       showName: true,
+      dynamicTitle: true,
```

Comment:
this adds the name next to the icon, making the buttons also more visible and easier to find (they're probably the only two buttons we care about)